base/card-api.gts

```diff
@@ -97,8 +97,9 @@ export class Card {
       return data;
     }
     let model = new this() as InstanceType<T>;
-    for (let [fieldName, value] of Object.entries(data)) {
-      serializedSet(model, fieldName, value);
+    let fields = getNonComputedFields(model);
+    for (let fieldName of Object.keys(fields)) {
+      serializedSet(model, fieldName, data[fieldName]);
```

Comment:
let's stay with the original behavior, and just record error documents in our index when instances have fields that don't exist